code/game/gamemodes/cm_self_destruct.dm

```diff
@@ -43,6 +43,7 @@ Regardless of where it's detonated, or how, a successful detonation will end the
 All of the necessary difines are stored under mode.dm in defines.
 */
 
+/*
```

Comment:
seems like everything is commented now, can just remove the file from DME and delete it